src/common/syncjournaldb.cpp

```diff
@@ -806,7 +807,17 @@ bool SyncJournalDb::updateErrorBlacklistTableStructure()
             sqlFail(QStringLiteral("updateBlacklistTableStructure: Add requestId"), query);
             re = false;
         }
-        commitInternal(QStringLiteral("update database structure: add errorCategory col"));
+        commitInternal(QStringLiteral("update database structure: add requestId col"));
+    }
+
+    if (columns.indexOf("remoteName") == -1) {
+        SqlQuery query(_db);
+        query.prepare("ALTER TABLE blacklist ADD COLUMN remoteName VARCHAR(4096);");
```

Comment:
Isn't there a central mechanism taking care of db migrations? This feels so wrong